src/SFA.DAS.AANHub.Application/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommandResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember
+{
+    public class PatchApprenticeMemberCommandResponse
+    {
+        public bool Sucess { get; set; }
```

Comment:
Misspelling here, should be `public bool Success { get; set; }`

Reply:
Misspelling has been rectified